### DISCUSSION TITLE
I need help with PTB

### DISCUSSION THREAD

**Discussion by Hedi2525** [python-telegram-bot/python-telegram-bot#4072] [Q&A]
I'm a beginner with Python-Telegram-Bot and i'm running this script without problems : 

```python
from telegram import Update
from telegram.ext import filters, MessageHandler, ApplicationBuilder, CommandHandler, ContextTypes

global motion
motion = 0

async def enable(update: Update, context: ContextTypes.DEFAULT_TYPE):
    await context.bot.send_message(chat_id=update.effective_chat.id, text="security is activated")
    
if __name__ == '__main__':
    application = ApplicationBuilder().token('My Token here......').build()
    
    enable_handler = CommandHandler('go', enable)
    application.add_handler(enable_handler)
        
application.run_polling()
```

My questions are  : How to get the variable "motion" equal 1 if the command "/go"  is sent to Telegram ?
After that how to delete this command "/go" from Telegram server ?

**Comment by Bibo-Joshi** [comment]
I recommend not to use global variables unless you have a very specific reason for that. PTB has a built-in solution for [storing data](../wiki/Storing-bot%2C-user-and-chat-related-data). For deleting messages, please have a look at [`Bot.delete_message`](https://docs.python-telegram-bot.org/en/stable/telegram.bot.html#telegram.Bot.delete_message).

**Reply by Hedi2525** [reply]
Thanks for answering.
Now I'm getting this error message when sending /get uuid :
key = context.args[0]
IndexError: list index out of range
